### DISCUSSION TITLE
Androidにてツイートボタンが機能しない

### DISCUSSION THREAD

**Discussion by akinov** [increments/qiita-discussions#477] [要望・アイデア・不具合]
このあたりの問題かと思われます。
https://twitter.com/mediatan/status/1604054778283073537
https://ba-artworks.com/tech/%E3%80%90html%E3%80%91%E5%AE%9F%E8%A3%85%E3%81%97%E3%81%9Ftwitter%E3%82%B7%E3%82%A7%E3%82%A2%E3%83%9C%E3%82%BF%E3%83%B3%E3%81%8C%E3%80%8C%E5%85%A5%E5%8A%9B%E3%81%97%E3%81%9F%E5%8D%98%E8%AA%9E%E3%81%AE/

**Comment by tanaka-asami415** [comment]
@akinov 
ご報告ありがとうございます！

ツイートボタンは、画像の赤い線で囲った箇所のもので間違いないでしょうか？

![image](https://github.com/increments/qiita-discussions/assets/103916185/76730efe-987f-44ae-9746-13c85207000c)


このボタンを押したときに下記エラーが出るというというということでしょうか？

> 「入力した単語の検索結果はありません。しばらくしてからやり直してください」とエラーが出る

詳細を教えていただけると幸いです🙏

**Reply by akinov** [reply]
詳細書けておらずすみません。
はい。そのボタンです。
Twitterアプリに遷移し↓の表示が出ます。

Android13
Twitterアプリは最新バージョンです。

![Screenshot_20230518-123848.png](https://github.com/increments/qiita-discussions/assets/1765937/0bbb34b3-04e8-4b49-98ad-7d0d2e6aac48)

**Reply by tanaka-asami415** [reply]
@akinov 
早速詳細をいただきありがとうございます🙏

弊社でもAndroid13で確認したのですが、同症状が発生せず
やはり貼っていただいた参考リンクの通り、環境が限られているようです。

修正は先になってしまうかもしれませんが、対応いたします！

**Comment by tanaka-asami415** [comment]
@akinov 
ご報告が遅くなってしまい申し訳ありません。
こちら10月に対応が完了しております！お手すきの際にご確認いただけますと幸いです🙏

**Reply by akinov** [reply]
ありがとうございます！動作確認できました！
